templates/forum/new_thread.html

```diff
@@ -13,10 +13,22 @@
 
 {% include 'forum/html_explanation.html' %}
 
-
-<table>
-{{form.as_p}}
-</table>
+<div>
+    {{ form.non_field_errors }}
+    {{ form.title.errors }}
+<p>
+    <label for="{{ form.title.id_for_label }}">Thread title:</label><br>
```

Comment:
could have been done as `{{ form.title.label_tag }}`, but I wanted to change the label text from 'title' to 'thread title'